package.json

```diff
@@ -25,7 +25,7 @@
     "async": "^3.1.0",
     "debug": "^4.1.1",
     "lodash": "^4.17.15",
-    "mongodb": "^3.6.1",
```

Comment:
This is why you were asking if we should make that a peer dep right? 😄

Reply:
Yup. I can update that in another PR though to keep things simple. (Maybe it'll require a major release?)